stable/external-dns/templates/secret.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or .Values.aws.assumeRoleArn (and .Values.aws.credentials.secretKey .Values.aws.credentials.accessKey) .Values.cloudflare.apiKey .Values.digitalocean.apiToken .Values.google.serviceAccountKey (and .Values.infoblox.wapiUsername .Values.infoblox.wapiPassword) .Values.rfc2136.tsigSecret .Values.extraEnv }}
+{{- if or .Values.aws.assumeRoleArn (and .Values.aws.credentials.secretKey .Values.aws.credentials.accessKey) .Values.cloudflare.apiKey .Values.digitalocean.apiToken .Values.google.serviceAccountKey (and .Values.infoblox.wapiUsername .Values.infoblox.wapiPassword) .Values.rfc2136.tsigSecret (and .Values.pdns.apiUrl .Values.pdns.apiKey) .Values.extraEnv }}
```

Comment:
Why `.Values.pdns.apiUrl`? You're not using that parameter on this file

Reply:
prob because there is no point in bothering w/ the apikey if the apiUrl is omitted

Reply:
That's the reason why we have the macro `external-dns.validateValues` defined in the **_helpers.tpl**. But I don't see any reason to do extra checks once you update the macro.